test/smoke/src/areas/task/task-quick-pick.test.ts

```diff
@@ -0,0 +1,75 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Application, Task, Terminal, TerminalCommandId } from '../../../../automation/';
+
+export function setup() {
+	describe('Task Quick Pick', () => {
```

Comment:
Lots of nesting here:

```
  VSCode Smoke Tests (Electron)
    Task
      Task Quick Pick
        run
          hide
            ✔ true
            ✔ false
            ✔ undefined
          icon
            - icon
            - color
            - icon & color
```

Proposal to align closer to what the terminal does:

```
  VSCode Smoke Tests (Electron)
    Task
      runTask command
          hide property - true
          hide property - false
          hide property - undefined
          icon - icon only
          icon - color only
          icon - icon & color
```